example/package.json

```diff
@@ -15,7 +15,9 @@
   "dependencies": {
     "react": "18.2.0",
     "react-native": "0.71.4",
-    "react-native-reanimated": "^3.0.2"
+    "react-native-reanimated": "^3.0.2",
```

Comment:
I think you need to upgrade `react-native-reanimate and react-native-gesture-handler` in `example` project

Reply:
updated